server/fixed_stops.json

```diff
@@ -0,0 +1,62 @@
+{
+  "San Jose": [
+    "2024-02-03T23:35:00Z",
+    "2024-02-03T00:35:00Z",
+    "2024-02-03T01:35:00Z",
+    "2024-02-03T02:38:00Z"
```

Comment:
also shouldnt we just store the times at which the train leaves? since storing the date we would never use

the entire json file could be these 6 lines
```json
[
    "23:35:00",
    "00:35:00",
    "01:35:00",
    "02:38:00"
]
```

Reply:
i did this because the times we get back from 511 is an entire date string, storing the times only definitely makes more sense though

i decided to store them in PST so it's less code, i can explain if you want but hopefully that's okay

resolved